app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -67,7 +67,7 @@ class MainActivity : AppCompatActivity() {
         }
     }
     fun setAppBarTitle(s:String){
-        supportActionBar!!.title=s;
+        supportActionBar!!.title="KeyBinder"
```

Comment:
Added a title to your app. Feel free to change to whatever name you would like.

Reply:
1) It looks like this is already passed in as a parameter... and is now hardcoded.
2) Avoid force unwrap (!!) at all costs.  That can create a NullPointerException, if the variable value is null.
Use something null safe, like foo?.let{} ?: instead.